ext/AvizExt/theme.jl

```diff
@@ -15,7 +15,7 @@ const COLORS::Dict{Symbol,Symbol} = Dict(
 )
 
 function colors(
-    scen_groups::Dict{<:Any,BitVector}
+    scen_groups::Dict{Symbol,BitVector},
```

Comment:
Why the change to `Symbol` here?

Reply:
Not sure might have been testing something and forgot about it, I'll revert

Reply:
Actually I think probably it makes more sense to change to Symbol, or there is something wrong here (I don't know this PR is the right place for that, tough). 

The return type of this function is a Dictionary when the keys are only allowed to be Symbols and they come from the keys of `scen_groups`, so this will raise an error if `scen_groups` has other type as keys... (right?)

Reply:
Yes @Zapiano that's right. I can't think of a case where we'd want non-symbol keys but we'd probably want to check first

Reply:
I'm going to work on Issue https://github.com/open-AIMS/ADRIA.jl/issues/549 now so I can take a look and change it in my PR, if that's ok with you.

Reply:
Thanks @Zapiano, sounds good :)